src/components/Data/SampleData.tsx

```diff
@@ -0,0 +1,37 @@
+export const SampleData = [
+  [
```

Comment:
I think the naming of this file path is confusing bc it's just a const and not a component, we should have another directory like `src/util` or something like that